etc/try_parser.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python
```

Comment:
I'm concerned about this huge custom parser for try strings, which is one reason I haven't really had a lot of time to review this change. Maybe we can chat live about ways of simplifying this change in order to make it more maintainable.

Reply:
One option would be to write parser with ply that we have in third_party for webidl parser, but I wanted standalone python script so it could also be run on edge services (ex. cloudflare workers).

Separate lexer could be merged in parser given that it's pretty simple syntax.